haskell_scotty/DBConfig.hs

```diff
@@ -0,0 +1,27 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module DBConfig (DBConfig(..), readDBConfig)
```

Comment:
@BartMassey I cargo-culted this `(..)` bit, and it won't compile without it. Can you explain why its needed?